src/components/calcite-tile/calcite-tile.scss

```diff
@@ -21,6 +21,14 @@
 :host([icon][heading]:not([description]):not([embed])) {
   @apply p-0;
 }
+:host(:not([input-alignment]):not([description])) .large-visual {
+  .icon {
+    @apply flex justify-center;
+  }
+  .content {
+    text-align: center;
+  }
+}
```

Comment:
I think all that is needed to center the icon in the "large visual" variant of Tile (ie., when there's no description passed in, only heading + icon) is this portion:

```scss
:host(:not([description])) .large-visual {
  .icon {
    @apply flex justify-center;
  }
}
```

(`input-alignment` refers to the Tile Select prop, not a prop within the Tile component.)

Reply:
@caripizza I added the text alignment for this scenario, but not sure if this would really be a use case in the real world. 
![CleanShot 2021-09-17 at 16 41 38](https://user-images.githubusercontent.com/9469422/133860933-787bbda8-1ec0-4404-beeb-70991e00027a.png)

Reply:
Ah didn't realize that, thanks for clarifying for me. In that case it seems like this may be better?

```scss
:host([icon][heading]:not([description])) {
  .large-visual {
    @apply text-center;
  }
}
```

Reply:
I'm not seeing the text-center style?

Reply:
Thought I had committed it up already, but I guess I removed it.